fluent-syntax/src/parser.js

```diff
@@ -242,6 +243,7 @@ export default class FluentParser {
 
     while (true) {
       ps.expectChar('\n');
+      ps.skipBlankLines();
       ps.skipInlineWS();
```

Comment:
This looks like a common pattern. Perhaps it's worth having a function called `expectIndent`?